spec/views/pages/home.html.erb_spec.rb

```diff
@@ -4,7 +4,8 @@
   describe "rendering home page" do
     it "contains \"Pages#home\"" do
       render
-      response.should have_text("Pages#home")
+        response.should have_text("Taste of code")
+        response.should have_text("Beginner coding workshop")
```

Comment:
Indentation of these two lines ^^ should be two spaces less, aligned with `render` 😉